docs/api/cli-options.md

```diff
@@ -76,6 +76,143 @@ Options include:
 
 <Callout variant="info" icon="💡">
 
-If you're using npm instead of yarn to publish Storybook, the commands work slightly different. For example, `npm run storybook build -- -o ./path/to/build`.
+The commands work slightly differently if you're using npm instead of Yarn to publish Storybook. For example, `npm run storybook build -- -o ./path/to/build`.
+
+</Callout>
+
+### `add`
+
+Installs a Storybook addon and configures your project for it. Read more in the [addon installation guide](../addons/install-addons.md).
+
+```shell
+storybook add [addon] [options]
+```
+
+Options include:
+
+| Option                     | Description                                                                                                                                                                     |
+| -------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `--package-manager`        | Sets the package manager to use when installing the addon.<br/> Available package managers include `npm`, `yarn`, and `pnpm`<br/>`storybook add [addon] --package-manager pnpm` |
+| `-s`, `--skip-postinstall` | Skips post-install configuration. Used only when you need to configure the addon yourself<br/>`storybook add [addon] --skip-postinstall`                                        |
+
+### `doctor`
+
+**Added in:** Storybook 7.6
```

Comment:
Why are we including these? Isn't that what versioned docs are for?

Reply:
That's exactly why I added this, so that when someone runs the command in a previous version (e.g., 7.5) and the command was only added in 7.6, this would fail as it's not present, wouldn't you agree?

Reply:
If someone is using `7.5`, we should assume they're reading `7.5` docs.

If we didn't make that assumption, we'd have to _massively_ restructure our docs on the whole. Since we're not going to do that, we should be consistent and also not do it here.